### DISCUSSION TITLE
How to set whisper.cpp with higher GPU priority?

### DISCUSSION THREAD

**Discussion by shadow-wxh** [ggml-org/whisper.cpp#2385] [Q&A]
I am running whisper.cpp as background service for a game however the game is using GPU as well and it is slowing whisper down. Is there a way to set whisper with higher GPU priority and let it finish computation first?

**Comment by thewh1teagle** [comment]
In windows you have option in the setting.
I wrote about it in https://github.com/thewh1teagle/vibe
Check the file install.md

**Comment by shadow-wxh** [comment]
This does improves a little bit however the strange thing is that once I turn on a screen recording app it speeds up the whisper even greater. I can not explain this, isn't that the more app running the more demanding for the GPU?

**Comment by shadow-wxh** [comment]
I still need someone explain to me why switching on a live streaming application is actually improve the process speed of whisper. Can It be done within the whisper c++ code itself?
